indy-besu/smart_contracts/demos/flow.ts

```diff
@@ -10,37 +10,29 @@ async function demo() {
   const faber = await new Actor().init()
   const alice = await new Actor().init()
 
-  console.log('1. Trustee creates DID Document')
-  receipt = await trustee.didRegistry.createDid(trustee.didDocument)
```

Comment:
Why have we removed DID creation here?

Reply:
Because we use a trustee that is already registered in the ledger, and its DID is derived from its address, attempting to restart the script will cause an error since this DID will already be registered by the previous script run

Reply:
and I think for the demo it’s not so important that the trustee creates a did